tests/LexAndYaccMiniProject/.gitignore

```diff
@@ -1,3 +1,4 @@
 Lexer.fs
 Parser.fs
-Parser.fsi
\ No newline at end of file
+Parser.fsi
+test.txt
```

Comment:
No, did not have to be a separate PR at all, that's totally fine! Thanks for the contribution!